templates/default/consul-template-systemd.erb

```diff
@@ -11,8 +11,8 @@ ExecStart=<%= @command %> <%= @options %>
 ExecReload=/bin/kill -HUP $MAINPID
 ExecStop=/bin/kill -INT $MAINPID
 KillMode=process
-Restart=on-failure
-RestartSec=42s
+Restart=always
+RestartSec=2s
```

Comment:
Probably best to let this be closer to 5 seconds? Give the system a little more time to recover.